### DISCUSSION TITLE
Support new datasets (local download)

### DISCUSSION THREAD

**Discussion by hboisgon** [Deltares/hydromt#729] [Feature requests]
This is a general issue to list and investigate interesting datasets to support with HydroMT (via update of deltares_data catalog).
You can edit this post directly to update the list or add below in comments.
To request a new dataset, add the name and link to where the dataset can be found.
For data available in public cloud or remote servers please use the separate issue #296 

Datasets (priority):

- [ ] New versions of E-OBS (already downloaded in hydromt-staging)
- [ ] copernicus esa_worldcover v200 - https://worldcover2021.esa.int/download (in hydromt staging)
- [ ] EURADCLIM - https://dataplatform.knmi.nl/dataset/rad-opera-hourly-rainfall-accumulation-euradclim-1-0
- [ ] OpenLandMap - https://www.openlandmap.org/ download at https://maps.elie.ucl.ac.be/CCI/viewer/download.php
- [ ] Harmonized World Soils Database HWSD 2.0  - https://gaez.fao.org/pages/hwsd
- [ ] meteorological stations (e.g. NOAA / WMO) - https://dev.meteostat.net/ - this is a package to access data, see if we can use directly the data itself

Other datasets:
- [ ] EDTM - https://zenodo.org/record/7634679#.Y__6nz1ByUk
- [ ] IRIS ICESat-2 River Surface Slope - https://zenodo.org/record/7516381
- [ ] GSDR-I - https://www.nature.com/articles/s41597-023-02238-4
- [ ] CHIRPS-GEFS: https://chc.ucsb.edu/data/chirps-gefs

**Comment by DirkEilander** [comment]
@hboisgon can you sort the list above in terms of priority based on datasets that you know will be used. Then @Tjalling-dejong can start adding some to the deltares_data catalog.

**Comment by hboisgon** [comment]
I tried my best to classify. Mainly new landuse maps and european meteo products. 
The soil maps from the FAO may be interesting to explore instead of soilgrids but would maybe require new developments in wflow so it could also wait.
Meteorological stations data (as well as discharge from GRDC actually, for now we only have locations) would also be nice and very useful for a lot of people I think. But the source is actually an API so we could also decide to add it when hydromt supports getting data via API. Else it might take a lot of time to track down if we can directly download the underlying data from all these sources.
I think the rest can wait (especially CHIRPS-GEFS which is a forecast product. If we decide to support forecast products I think better to use API as content will always be changing)

**Comment by DirkEilander** [comment]
From my side there is the 
- [ ] emodnet (bathymetry for europe in hydromt_staging folder)
- [ ] mdt_cnes_cls22 (update from mdt_cnes_cls18; also in hydromt_staging)
